addons/web_dashboard/controllers/main.py

```diff
@@ -0,0 +1,31 @@
+# -*- coding: utf-8 -*-
+
+from openerp import http
+from openerp.http import request
+
+
+class Dashboard(http.Controller):
+
+    @http.route('/dashboard/info', type='json', auth='user')
+    def get_info(self, **kw):
+        installed_apps = request.env['ir.module.module'].search_count([('application', '=',  True), ('state', 'in', ['installed', 'to upgrade', 'to remove'])])
+        active_users = request.env['res.users'].search_count([('active', '=', True), ('login_date', '!=', False)])
```

Comment:
Invited user by default having access rights of `Template User` (auth_signup.template_user_id). So i included portal user as a active users.